app/views/application_letters/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <%- model_class = ApplicationLetter -%>
 <div class="page-header">
-  <h1><%=t '.title', :default => model_class.model_name.human.pluralize.titleize %></h1>
+  <h1><%=t '.title', :default => model_class.model_name.human(:count => model_class.count).titleize %></h1>
```

Comment:
Well, it should be a new issue.

Our issue originally only concerned the event overview page and the application letter detail page. Then we also got a bug issue to turn Berwerbungs into the correct Bewerbungen which does concern the application letter index page (as in if have the correct prural in our data anyway we can also use it in this page), but that issue was also not about restructuring the application letter index page.